### ISSUE TITLE
Add favorites

### ISSUE THREAD

**Issue by ishan0102** [ishan0102/engblogs#35]
Bottom left hand side, save to local storage, separate page called /favorites maybe

**Comment by raghunandanbhat** [comment]
Is it something like bookmarking a post and show it on /favorites page?

**Comment by ishan0102** [comment]
> Is it something like bookmarking a post and show it on /favorites page? 

Yeah basically, just trying to imitate what we have on the mobile version

![image](https://github.com/ishan0102/engblogs/assets/47067154/cd3c0140-7efb-4b76-aeae-a1d9b17b8ad5)

**Comment by raghunandanbhat** [comment]
do you have user accounts in mobile version or that's also IP based?

**Comment by ishan0102** [comment]
haha so that's a funny hack, @mxchen2001 generated UUIDs and stored them as IPv6 addresses bc you can't pull IPs out on mobile.

@mxchen2001 should we add ios as a folder to this repo or do you want to keep that closed
